### PR TITLE
Hackathon. Cronjob to collect data for PowerBI integration.

### PR DESCRIPTION
**Pull request by thomasvn**
## What does this PR change?

Currently testing using the following `values-integrations.yaml`:

```yaml
# helm template ./cost-analyzer -f values-integrations.yaml -s templates/data-export-postgres-cronjob-template.yaml > integrations.yaml

integrations:
  postgres:
    enabled: true
    imagePullSecrets: my-docker-key
    fullImageName: docker.io/mclean0328/kube-postgres-cron:latest
    kubecostEndpoint: "https://kubecost.me.com/model/allocation"
    databaseName: "postgres"
    databaseUser: "test"
    databasePassword: "test"
    databaseHost: "my.postgres.database.azure.com"
    databasePort: "5432"
    schedule: '0 * * * *'

```

TODO:
- Template needs sane defaults. Or template needs to throw error if user does not provide specific configs.
- Template needs to include metadata + labels similar to other templates in this repo.
- Update script to parse arguments. Based on the argument, run a different data export job.

### CODE REVIEW

File: cost-analyzer/values.yaml [hunk 1]
```diff
@@ -1191,6 +1191,21 @@ costEventsAudit:
 ##
 # readonly: false
 
+## Kubecost Integrations
+## 
+integrations:
+  powerbi:
```
Comment (mclean0328) [inline cost-analyzer/values.yaml]: Wondering if the integration should just be called "postgres". The example we are working in terms of end to end pipeline is getting the data somewhere that powerbi can connect to pull it, but ultimately the job that runs is just going to be sending data to postgres and has nothing to do with powerbi

File: cost-analyzer/values.yaml [hunk 2]
```diff
@@ -1191,6 +1191,21 @@ costEventsAudit:
...
+    enabled: true
+    imagePullSecrets: my-docker-key
+    fullImageName: docker.io/mclean0328/test-cron:latest
+    kubecostEndpoint: "https://kubecost.me.com/model/allocation"
```
Comment (mclean0328) [inline cost-analyzer/values.yaml]: The example kubecost endpoint should not include /model/allocation. The script takes that into account. So https://kubecost.me.com/ works just fine for the example.

File: cost-analyzer/templates/powerbi-cronjob-template.yaml [hunk 3]
```diff
@@ -0,0 +1,54 @@
+{{ if .Values.integrations.powerbi.enabled }}
```
Comment (mclean0328) [inline cost-analyzer/templates/powerbi-cronjob-template.yaml]: The template overall looks good. Just potentially changing all the 'powerbi' references in the values to 'postgres' if we think calling the integration 'postgres' is better

File: cost-analyzer/values.yaml [hunk 4]
```diff
@@ -1191,6 +1191,21 @@ costEventsAudit:
 ##
 # readonly: false
 
+## Kubecost Integrations
+## 
+integrations:
+  powerbi:
+    enabled: true
+    imagePullSecrets: my-docker-key
+    fullImageName: docker.io/mclean0328/test-cron:latest
```
Comment (mclean0328) [inline cost-analyzer/values.yaml:1301]: Update the docker image to this "docker.io/mclean0328/kube-postgres-cron:latest "

File: cost-analyzer/templates/powerbi-cronjob-template.yaml (hunk 3 above)
Comment (thomasvn) [reply cost-analyzer/templates/powerbi-cronjob-template.yaml]: Updated!

File: cost-analyzer/values.yaml (hunk 1 above)
Comment (thomasvn) [reply cost-analyzer/values.yaml]: Updated!

File: cost-analyzer/values.yaml (hunk 4 above)
Comment (thomasvn) [reply cost-analyzer/values.yaml:1301]: Updated!

As a TODO, we should try to get this image into the `kubecost1` or `guestbook` container registry.

File: cost-analyzer/values.yaml (hunk 2 above)
Comment (thomasvn) [reply cost-analyzer/values.yaml]: Updated!

**Pull request closed without merging**